charts/sysdig/README.md

```diff
@@ -109,6 +109,7 @@ The following table lists the configurable parameters of the Sysdig chart and th
 | `slim.resources.requests.cpu`                                        | CPU requested for building the kernel module                                             | `1000m`                                                                        |
 | `slim.resources.requests.memory`                                     | Memory requested for building the kernel module                                          | `348Mi`                                                                        |
 | `slim.resources.limits.memory`                                       | Memory limit for building the kernel module                                              | `512Mi`                                                                        |
+| `slim.env`                                                           | Environment variables for the kernel module image builder. Provide as map of `VAR: val`  | `{}`                                                                           |
```

Comment:
Since it's specific to kmodule, I'd like it better to have the name reflect it, otherwise one could expect them to be applied to the slim container too. Or else we could just actually apply them to both.